avocado_vt/test.py

```diff
@@ -217,28 +218,48 @@ def setUp(self):
         Avocado-vt uses custom setUp/test/tearDown handling and unlike
         Avocado it allows skipping tests from any phase. To convince
         Avocado to allow skips let's say our tests run during setUp
-        phase and don't do anything during runTest/tearDown.
+        phase and report the status in test.
         """
         env_lang = os.environ.get('LANG')
         os.environ['LANG'] = 'C'
         try:
             self._runTest()
+            self.__status = "PASS"
         # This trick will give better reporting of virt tests being executed
         # into avocado (skips, warns and errors will display correctly)
-        except error.TestNAError, details:
-            raise exceptions.TestSkipError(details)
-        except error.TestWarn, details:
-            raise exceptions.TestWarn(details)
-        except error.TestError, details:
-            raise exceptions.TestError(details)
-        except error.TestFail, details:
-            raise exceptions.TestFail(details)
+        except:  # Old-style exceptions are not inherited from Exception()
```

Comment:
Well people should not use the `avocado` exceptions directly from `avocado-vt`. Anyway if you think this is needed I can add this exception.

The thing with exceptions is that they are part of the stable API in `autotest`, but in `avocado` they are part of `core` and can (and do) change in time. People should never import from `avocado.core` in tests, they should use `self.error`, `self.fail` and so on. In `avocado-vt` we do the stable mapping from `autotest.client.shared.error` exceptions to `avocado` exceptions so people should use those.